java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3918,6 +3921,62 @@ public <ArgType> T xpending(
         return getThis();
     }
 
+    /**
+     * Returns information about the stream stored at key <code>key</code>.<br>
+     * To get more detailed information use {@link #xinfoStreamFull(ArgType)} or {@link
+     * #xinfoStreamFull(ArgType, int)}.
+     *
+     * @implNote ArgType is limited to String or GlideString, any other type will throw
```

Comment:
Please copy this from #1820 for all commands. It has fixed format there.

Reply:
fixed